packages/web-pixel-ui-extensions/README.md

```diff
@@ -0,0 +1,28 @@
+# Web Pixel Extension
+
+Provides an `extend` method to register a Web Pixel Extension.
+
+## Usage
+
+Registering a Web Pixel Extension:
+
+```js
+import {extend} from '@shopify/web-pixel-ui-extensions';
+
+extend('WebPixel::Render', ({analytics, context, config}) => {
```

Comment:
Do we think there will be more than one extension point for web pixels? I like that under the hood this uses the same registration setup as the other extension types, but I feel like it is unnecessarily verbose for consumers if they are likely to only ever implement one extension point. I wonder if it would be better to just get closer to what I am proposing in https://github.com/Shopify/ui-extensions-private/discussions/1653#discussioncomment-1927495, and use a signature that does not include the extension point, since in your case only one is relevant. Something like this is what I was imagining as a bridge to the world where we just use exports to denote extension registrations:

```ts
import {register} from '@shopify/web-pixels';

register(({analytics, context, config}) => {
  // do your thing in here
});
```

Reply:
So the only reason I went with `extend` was to follow the pattern and the rational was to bring consistency in the developer experience. If you think that having a different way of registering the Web Pixel Extension is acceptable then I'm on board.

Reply:
Prefer the `register()`-API compared to `extend()` here too, unless we'll end up with multiple extension points for web pixels. The way I understand it, is that web pixels are not tied to a specific layout point and registering them is more of a "global" thing. For layout points the `extend()`-API makes more sense.

Reply:
`register` is closer to what the developer is actually doing so I'm good with renaming `extend`.  Would this mean we'd support calling `register` with or without an extension point?

Reply:
@vividviolet for Pixels, `register` should not take an extension point, since there is only one extension. In extensions, I think we should introduce this function alongside the "list your extension point => module mapping in your configuration file, we will do the rest" change in my proposal, which would mean it would also not accept an extension point identifier.

Reply:
No, I don't think so. `register` in a vanilla extension could be omitted entirely when we move to using exports to declare registrations:

```ts
import {register} from '@shopify/checkout-ui-extensions';

function extension(root, {extensionPoint}) {
  root.appendChild(root.createText(`Rendering in ${extensionPoint}`));
}

// Equivalent
export default extension;
export default register(extension);
```

Having the hook of `register` lets us give better typing hints, and a hook for doing framework-specific bindings, e.g.:

```ts
import {registerReact} from '@shopify/checkout-ui-extensions-react';

export default registerReact(({extensionPoint}) => <>Rendering in {extensionPoint}</>);
```